setup.py

```diff
@@ -192,5 +191,6 @@ def run(self, *args, **kwargs):
                      'pythran-config = pythran.config:run']},
       setup_requires=["pytest-runner"],
       tests_require=['pytest', 'pytest-pep8'],
+      extras_require={'deps': ['numpy']},
```

Comment:
It would work:
- if the user were to run `pip install -e .` which should install `numpy` using pip

and would not:
- if the user runs `python setup.py install` or `python setup.py develop` where easy_install is used.

I added it as an extra dep. because I assumed [you made this decision with good intentions](
https://github.com/ashwinvis/pythran/blob/7df4daa8011d4832bafbd54cc0b53b19ac052fb1/setup.py#L16). The problem arises when numpy is installed with `easy_install` yes?

Alternatively numpy can be added to the `install_requires` argument, which is processed when `pip` runs.